x-pack/filebeat/module/tomcat/log/test/generated.log-expected.json

```diff
@@ -0,0 +1,5522 @@
+[
+    {
+        "@timestamp": "2016-01-29T08:09:59.000Z",
+        "event.code": "asdf",
+        "event.dataset": "tomcat.log",
+        "event.module": "tomcat",
+        "event.original": "%APACHETOMCAT-1516-asdf: 10.251.224.219||eacommod||rci||[29/Jan/2016:6:09:59 OMST]||exercita||https://example.com/illumqui/ventore.html?min=ite#utl||vol||amremap||oremi||ntsunti||5293||https://mail.example.net/turadipi/aeca.htm?ntium=psaq#cer||Mozilla/5.0 (Linux; Android 9; G8142) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.83 Mobile Safari/537.36||aliqu",
+        "event.timezone": "OMST",
+        "fileset.name": "log",
+        "http.request.referrer": "https://mail.example.net/turadipi/aeca.htm?ntium=psaq#cer",
+        "http.response.body.content": "vol",
```

Comment:
I'm assuming this also isn't the actual http content, so I think we should just drop it